static/css/starter-template.css

```diff
@@ -57,7 +60,7 @@ video {
   text-align: center;
   font-size: 2rem;
   padding-top: .3em;
-  color: #3758ce;
+  color: #3f6695;
```

Comment:
Are these foundation colour changes? Or was there another reason to change this colour?

Reply:
This is to keep the button colour inline with the menu colour for the upstream repository

Reply:
The help file uses https://github.com/bbcmicrobit/PythonEditor/blob/master/static/css/starter-template.css hence why i copied the button style over. Not sure why it's used over style.css, but currently if you import style.css it breaks the scroll on the page and has a bunch of other styling issues to fix so probably requires some extra work to unify this into a single file.

Reply:
I see, so the `starter-template.css` is only used in the help file?
I guess the easiest way to deal with that is to keep as them as separate files rather than isolate the styles in `styles.css` that is changing global elements.
We'll need to make sure keep this in mind when pulling into the foundation repo.